### DISCUSSION TITLE
Fine tune wholeBrainSeg_Large_UNEST_segmentation model

### DISCUSSION THREAD

**Discussion by qwang0225** [Project-MONAI/MONAI#7355] [Q&A] [answered]
Hi, I'm trying to segment insula on an MRI dataset of stroke patients using  wholeBrainSeg_Large_UNEST_segmentation model from monai zoo. I'm faced with two challenges.
1. How to make the model produce 5 classes(background, left/right pos and ant insula) instead of all 133 classes?
2. How to fine tune the model on my stroke dataset, assuming insula labels have been produced by manual segmentation.
Any input would be appreciated! Thanks.

**Comment by KumoLiu** [comment, marked as answer]
Hi @qwang0225, you can refer to this tutorial.
https://github.com/Project-MONAI/tutorials/tree/main/model_zoo
Hope it helps, thanks!

**Reply by qwang0225** [reply]
Thanks a lot! I'll check it out.
